scripts.js

```diff
@@ -1 +1,16 @@
-console.log('Hello World!');
\ No newline at end of file
+function getGreeting(name, surname) {
+    console.log(`Hi ${name} ${surname}! Good to see you`)
+}
+const greeting = getGreeting('John', 'Smith');
+
+console.log(greeting);
```

Comment:
<img width="574" alt="Screenshot 2023-08-28 at 18 16 51" src="https://github.com/szymon-czudowski/WANACODE-javascript-basics-homework/assets/29953630/15951733-ba63-4b0e-b036-b26a8547b3e4">
I don't think this console.log() will return what is expected on the screenshot, it will log undefined. Do you know why is that?